prov/shm/src/smr_av.c

```diff
@@ -123,6 +133,8 @@ static int smr_av_insert(struct fid_av *av_fid, const void *addr, size_t count,
 			smr_map_to_endpoint(smr_ep->region, shm_id);
 			smr_ep->region->max_sar_buf_per_peer =
 				SMR_MAX_PEERS / smr_av->smr_map->num_peers;
+			srx = smr_get_peer_srx(smr_ep);
+			srx->owner_ops->foreach_unspec_addr(srx, &smr_get_addr);
```

Comment:
This change will cause a segfault in efa, because efa hasn't implement such op

Reply:
It's a little tricky here. We would have to implement such op in efa, efa in main branch hasn't onboarded util srx. We may have to make it as a no-op for this PR. The peer API ops in efa provider are defined in https://github.com/ofiwg/libfabric/blob/main/prov/efa/src/rdm/efa_rdm_srx.c. I can open a PR to add this as a no-op if you prefer that.

Reply:
I opened a PR https://github.com/ofiwg/libfabric/pull/8998 to add this op as no-op. You need to rebase on this PR after its merged.